stellargraph/layer/gcn_lstm.py

```diff
@@ -212,41 +195,41 @@ def call(self, features):
 class GraphConvolutionLSTM:
 
     """
-        A stack of 2 Graph Convolutional layers followed by an LSTM, Dropout and,  Dense layer.
+        A stack of N1 Graph Convolutional layers followed by N2 LSTM, Dropout and,  Dense layer.
```

Comment:
I think this might read a bit clearer:

```suggestion
        A stack of N1 Graph Convolutional layers followed by N2 LSTM layers, a Dropout layer, and a Dense layer.
```